schema/pgsql/schema.sql

```diff
@@ -243,14 +245,15 @@ CREATE TABLE event (
     id bigserial,
     time bigint NOT NULL,
     object_id bytea NOT NULL REFERENCES object(id),
-    type event_type NOT NULL,
+    type event_type,
     severity severity,
     message text,
     username citext,
     mute boolenum,
     mute_reason text,
 
-    CONSTRAINT pk_event PRIMARY KEY (id)
+    CONSTRAINT pk_event PRIMARY KEY (id),
+    CHECK (type IS NOT NULL)
```

Comment:
Does this make any difference in behavior? To me, this just looks like it makes the schema definition more complex.

Reply:
It makes the schemas more equal looking to reduce confusion. I'm worried specifically about someone seeing the column definition difference and trying to "fix" the "missing" NOT NULL in MySQL (while overseeing the CHECK). This would cause a regression.